lib/workers/repository/extract/index.ts

```diff
@@ -56,5 +56,22 @@ export async function extractAllDependencies(
     }
   }
   logger.debug(`Found ${fileCount} package file(s)`);
+
+  // If enabledManagers is non-empty, check that each of them has at least one extraction.
+  // If not, log a warning to indicate possible misconfiguration.
+  if (is.nonEmptyArray(config.enabledManagers)) {
+    for (const enabledManager of config.enabledManagers) {
+      if (
+        !(enabledManager in extractions) ||
+        extractions[enabledManager].length === 0
```

Comment:
I don't think a manager will get added unless the length of package files is greater than 0?

Reply:
Hi @rarkins I've checked and a manager really gets added even if zero package files found:

For example, here is how the `npm` entry looks like in the `extractions` object:
```
{ npm: [ {} ] }
```

So it seems my second condition would be needed...